src/_data/platforms.yml

```diff
@@ -93,10 +93,10 @@
       support_level: production
       type: framework
       name: Android
-      doc_link: /clients/java/modules/android/
+      doc_link: /platforms/android/
       wizard_parent: java
       wizard:
-        - _documentation/clients/java/integrations.md#android
+        - _documentation/platforms/android/index.md
```

Comment:
this file didn't have any changes, and afaik, we'd need to add something like `<!— Wizard android —>` into it.
See: https://github.com/getsentry/sentry-docs/pull/1498#discussion_r378828245
so that's the way of how the wizard would extract the right piece of code from the whole md file